code/modules/reagents/reagent_dispenser.dm

```diff
@@ -3,8 +3,8 @@
 	desc = "..."
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "watertank"
-	density = 1
-	anchored = 0
+	density = TRUE
+	anchored = TRUE
```

Comment:
This change should go in the CL.

Reply:
> This change should go in the CL.

Will do when I push another update

Reply:
Should still be in the CL. The examine text is a good extra though